File: Samples/CSSample/Procotol/ShipAckMsgHandler.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.InteropServices;
using DOSSystem;
using DOSSample;


namespace DOSSample
{
	
	public class CShipAckMsgHandler
	{
		IShipAck	m_pMsgHandler;
		public void InitMsgMap(IShipAck pMsgHandler,Dictionary<uint, DistributedObject.MsgHandler> MsgMap)
		{
			m_pMsgHandler = pMsgHandler;
			
		}

		
	};
};

File: Samples/JSSample/src/Protocol/ShipAckMsgHandler.ts
import { CSmartValue, CSmartArray, CSmartStruct } from "../DOSSystem/SmartStruct";
import { DOSMakeMsgID, DOS_MSG_MAP, OBJECT_ID } from "../DOSSystem/DOSSystem";
import { MODULE_IDS, DOSSAMPLE_INTERFACES } from "./DOSSampleSystemEnums";
import { IShipAck } from "./ShipAckInterface";
import { SHIP_METHODS } from "./ShipInterface";


export class CShipAckMsgHandler
{
	protected m_pMsgHandler: IShipAck;
	public InitMsgMap(pMsgHandler: IShipAck,MsgMap: DOS_MSG_MAP): void
	{
		this.m_pMsgHandler=pMsgHandler;
		
	}

	
};
